bin/sidecli.re

```diff
@@ -118,19 +118,22 @@ let info_create_transaction = {
 };
 
 let create_transaction = (sender_wallet_file, received_address, amount) => {
+  open Networking;
+  let.await validators_uris = validators_uris();
+  let validator_uri = List.hd(validators_uris);
+  let.await block_level_response = request_block_level((), validator_uri);
+  let block_level = block_level_response.level;
   let.await wallet = Files.Wallet.read(~file=sender_wallet_file);
   let transaction =
     Operation.Side_chain.sign(
       ~secret=wallet.priv_key,
       ~nonce=0l,
-      ~block_height=0L,
+      ~block_height=Int64.(add(block_level, of_int(100))),
```

Comment:
Probably you misread, https://github.com/marigold-dev/sidechain/pull/68#discussion_r677777268

This should be just

```suggestion
      ~block_height=block_level,
```